src/utils/errorHandler.js

```diff
@@ -18,6 +21,13 @@ export function handleError(event, options = {}) {
     console.log(event);
   }
 
+  if (event && event.message === NETWORK_CONNECTION_ERROR) {
+    options = {
+      title: NETWORK_CONNECTION_ERROR,
+      message: NETWORK_ERROR_MESSAGE,
```

Comment:
Let's only change the message below and not use the `options`. Let it have the default `title`.

Reply:
done as suggested. Only changed the `message` variable.